ntoskrnl/mm/section.c

```diff
@@ -3663,6 +3663,7 @@ MiRosUnmapViewOfSection(
         PMM_IMAGE_SECTION_OBJECT ImageSectionObject;
         PMM_SECTION_SEGMENT SectionSegments;
         PMM_SECTION_SEGMENT Segment;
+        ULONG MapCount = 0;
```

Comment:
This is more like a stylish-only suggestion, but nevertheless.
Is it necessarily to set `MapCount` to zero here? It's always overwritten below, at line 3704, on `InterlockedDecrement()` call anyway. Also it's not used anywhere above that line, as I see, so initializing it here simply does not do anything.
I suggest to keep it as follows:
```suggestion
        ULONG MapCount;
```
P. S.: this is what Mark already suggested below (together with `ImageBaseAddress = NULL` suggestion, I'm agreed with that too). However, my suggestion looks more correct according to out coding style guidelines I think.

Reply:
@oleg-dubinskiy, Changed. Thanks.